msteams-platform/bots/what-are-bots.md

```diff
@@ -46,17 +46,24 @@ You can use the Teams bot APIs to get information for members of a chat or team.
 <!--- TBD: For quick scanning, see if the above information can be itemized as a list.
 --->
 
-## Next step
+## Global endpoint for bots API
 
-> [!div class="nextstepaction"]
-> [Bots and SDKs](~/bots/bot-features.md)
+When creating a bot, the global routing endpoint can be enabled by setting the `cloud` parameter as `teams` in the URL. For example, the URL should be `https://smba.trafficmanager.net/teams/v3/conversations/12345`.
```

Comment:
> When creating a bot

What does this mean? The "cloud" isn't set on bot creation, is it? It's something that's on every request to SMBA.